src/tests/unit/tests/injected/components/command-bar.test.tsx

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { Mock, Times } from 'typemoq';
+
+import { BaseButton, Button } from '../../../../../../node_modules/office-ui-fabric-react';
+import { CopyIssueDetailsButton } from '../../../../../common/components/copy-issue-details-button';
+import { UserConfigurationStoreData } from '../../../../../common/types/store-data/user-configuration-store';
+import { CommandBar, CommandBarDeps, CommandBarProps } from '../../../../../injected/components/command-bar';
+import { DecoratedAxeNodeResult } from '../../../../../injected/scanner-utils';
+import { BaseDataBuilder } from '../../../common/base-data-builder';
+import { EventStubFactory } from '../../../common/event-stub-factory';
+
+describe('CommandBar', () => {
+    describe('renders', () => {
+        const showInspectButtonMessage = [true, false];
+
+        it.each(showInspectButtonMessage)('renders, shows inspect button message: %s', show => {
+            const props = defaultCommandBarPropsBuilder()
```

Comment:
>defaultCommandBarPropsBuilder [](start = 26, length = 29)

I'm not sure I understand the need for the builder function here; it feels like an extra layer of overhead compared to:

```ts
const props: CommandBarProps = {
    ...defaultCommandBarProps, // or ...defaultCommandBarProps(), if you'd like to keep the separate-object-per-test guarantee
    shouldShowInspectButtonMessage: () => show
}
```

the object expansion version also plays nicer with F2 renaming of the property being overridden.

Reply:
make sense